src/app/features/pictours/index.js

```
@@ -0,0 +1,242 @@
+      .addCase(getPicTourCategories.fulfilled, (state, action) => {
+        state.isFetching = false;
+        state.categories = action?.payload;
+      })
+      .addCase(getPicTourCategories.rejected, (state, action) => {
+        state.isFetching = false;
+        state.error = action?.payload;
+      })
+      .addCase(getPicTourByCategory.pending, (state) => {
+        state.isPicTourFetching = true;
+      })
+      .addCase(getPicTourByCategory.fulfilled, (state, action) => {
```

ESLint

[error] 'action' is defined but never used.

(no-unused-vars)